pkgs/by-name/mo/monophony/package.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, fetchFromGitLab
+, python3Packages
+, wrapGAppsHook4
+, gst_all_1
+, gobject-introspection
+, gtk4
+, yt-dlp
```

Comment:
```suggestion
```
using `buildPythonPath` as suggested below removes this requirement.

Reply:
Okay

Reply:
Yt-dlp wouldnt be installed at all then

Reply:
I don't install yt-dlp as a python package, and neither does the AUR script

Reply:
oh, my bad. i thought it was listed in the python3Packages. if it is not in the python3Packages will have to disregard my comment about changing the --prefix PATH as buildPythonPath will not find it.

Reply:
Okay